ES/ESmy020.xml

```diff
@@ -31,71 +31,394 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                               <depth unit="mm">40</depth>
                            </dimensions>
                         </extent>
                         <foliation><!--2016-10-18+02:00 Updated MY-020 from DomLib--></foliation>
                         <collation>
+                          <note></note>
                            <list>
-                              <item xml:id="q1">
-                                 <locus from="1r" to="6v"/>I(fols. 1r-6v) </item>
-                              <item xml:id="q2">
-                                 <locus from="7r" to="14v"/> II(fols. 7r-14v) </item>
-                              <item xml:id="q3">
-                                 <locus from="15r" to="22v"/> III(fols. 15r-22v) </item>
-                              <item xml:id="q4">
-                                 <locus from="23r" to="33v"/> IV(fols. 23r-33v) </item>
-                              <item xml:id="q5">
-                                 <locus from="34r" to="43v"/> V(fols. 34r-43v) </item>
-                              <item xml:id="q6">
-                                 <locus from="44r" to="53v"/> VI(fols. 44r-53v) </item>
-                              <item xml:id="q7">
-                                 <locus from="54r" to="71v"/> VII(fols. 54r-71v)&#xD; &#xD; Quire I
-                                 is composed entirely of fols. from other parts of the Ms., its
-                                 folios are attached as guard leaves by means of crude stitches. All
-                                 the folios of quire I belong to Text I.</item>
+                              <item xml:id="q1" n="1">
+                                 <locus from="1r" to="5v"/>
+                                 <dim unit="leaf">4</dim>
+                                  <locus from="1r" to="14v"/>
+                                  s.l.: 1, no stub
+                                  s.l.: 2, no stub
+                                  s.l.: 3, no stub
+                                  s.l.: 4, no stub
+                                   </item>
+                              <item xml:id="q2" n="2">
+                                 <locus from="6r" to="7v"/>
+                                 <dim unit="leaf">2</dim>
+                                  <dim unit="leaf">2</dim>
+                                </item>
+                              <item xml:id="q3" n="3">
+                                 <locus from="6r" to="7v"/>
+                                 <dim unit="leaf">2</dim>>
+                                 <locus from="15r" to="22v"/>
+                                 <dim unit="leaf">8</dim>
+                               </item>
+                              <item xml:id="q4" n="4">
+                                 <locus from="23r" to="33v"/>
+                                  <dim unit="leaf">11</dim>
+                                  s.l.: 11, stub after 10
```

Comment:
this is a problem for the quire analysis, in a true quire leaf 11 may only have stub before 1, or it is an added singleton

Reply:
ok, redone